models/baseModels/SalesQuote/SalesQuote.ts

```diff
@@ -48,6 +55,18 @@ export class SalesQuote extends Invoice {
     return invoice;
   }
 
+  static filters: FiltersMap = {
+    numberSeries: (doc: Doc) => ({ referenceType: doc.schemaName }),
+  };
+
+  async afterSubmit(): Promise<void> {
+    await super.afterSubmit();
+    if (this.referenceType == ModelNameEnum.Lead) {
+      const leadData = (await this.loadAndGetLink('party')) as Lead;
+      await leadData.setAndSync('status', 'Quotation');
```

Comment:
variable name should be identical to the doctype. also wrong assertion as `Lead`